packages/protocol/contracts/L1/gov/TaikoGovernor.sol

```diff
@@ -124,6 +124,23 @@ contract TaikoGovernor is
         return 1_000_000_000 ether / 10_000; // 0.01% of Taiko Token
     }
 
+    /// @dev Cancel a proposal with GovernorBravo logic.
```

Comment:
We have an issue with contract size.

> `TaikoGovernor` is above the contract size limit (25229 > 24576).

Not sure how we can get round of this issue.

Reply:
Asked OZ.

Reply:
Also applied (locally) what i suggested, and still above the threshold..

Reply:
Ok, i have a workaround for the size issue : https://github.com/taikoxyz/taiko-mono/pull/16363